index.css

```diff
@@ -61,4 +61,4 @@ body {
     border-color: gold;
     min-width: 300px;
   }
-  
\ No newline at end of file
+  
```

Comment:
Kindly remove the excess line at the end of your CSS code. This is to make sure that it passes the stylelint error.